challenges/dailyRevenue/daily-revenue.js

```diff
@@ -0,0 +1,15 @@
+module.exports = timestamps => {
+  return timestamps.reduce((acc, curr) => {
+    // create date string from timestamp
+    const date = new Date(curr.timestamp).toDateString();
```

Comment:
be careful. this will give a different result in oregon compared to new york. You need to use utc time to normalize on timezone